python/spacewalk/spacewalk-backend.changes

```diff
@@ -1,3 +1,9 @@
+- Exclude invalid mirror urls for reposync (bsc#1203826)
```

Comment:
The format of the entry is changing on releasing the package. There is no need to format it this way, just keep it as a one level entry.
Something like:
```
- Use only plain text as mirrorlist (bsc#1203826)
```
as XML is relevant for metalinks and this change is not affecting this part if I'm not wrong.